stag-library-compiler/src/main/java/com/vimeo/stag/processor/utils/TypeUtils.java

```diff
@@ -456,26 +456,13 @@ public static boolean isNativeArray(@NotNull TypeMirror type) {
         return (type instanceof ArrayType);
     }
 
-    /**
-     * Method to check if the {@link TypeMirror} is of {@link Collection} type
-     *
-     * @param type :TypeMirror type
-     * @return boolean
-     */
-    public static boolean isSupportedCollection(@Nullable TypeMirror type) {
-        return type != null && (isNativeArray(type) || isSupportedList(type));
-    }
-
     /**
      * Method to check if the {@link TypeMirror} is of {@link List} type
      *
-     * @param type :TypeMirror type
+     * @param type TypeMirror type
      * @return boolean
```

Comment:
optional: you could change the 4 `@return boolean` docs in this class to be more descriptive instead of just declaring the return type.  Eg: `@return true if the given type is a List, otherwise false`.